crates/debug/src/transform/expression.rs

```diff
@@ -47,7 +53,10 @@ impl Clone for CompiledExpressionPart {
     fn clone(&self) -> Self {
         match self {
             CompiledExpressionPart::Code(c) => CompiledExpressionPart::Code(c.clone()),
-            CompiledExpressionPart::Local(i) => CompiledExpressionPart::Local(*i),
+            CompiledExpressionPart::Local { label, trailing } => CompiledExpressionPart::Local {
```

Comment:
Tangent: Why aren't we doing `#[derive(Clone)]` for `CompiledExpressionPart`? I don't see anything here that is different from what the derive would do.

Reply:
Not sure. But it is gone: we have `[derive(Clone)]` now.